tests/integration_tests/conftest.py

```diff
@@ -188,7 +188,11 @@ def remove(self) -> None:
 
 
 def setup_presto_if_needed():
-    backend = app.config["SQLALCHEMY_EXAMPLES_URI"].split("://")[0]
+    db_uri = (
+        app.config.get("SQLALCHEMY_EXAMPLES_URI")
+        or app.config["SQLALCHEMY_DATABASE_URI"]
+    )
+    backend = db_uri.split("://")[0]
```

Comment:
this seems like an unrelated change, no?

Reply:
It's the related change. When you want to run local test with `SQLALCHEMY_EXAMPLES_URI = None` you will have an exception here and I fix the issue.

> URI to database storing the example data, points to
SQLALCHEMY_DATABASE_URI by default if set to `None`